ToolChain/Assembler/Parser/Parser.py

```diff
@@ -605,7 +429,6 @@ def translateTextImmediateToImmediate(self, textImmediate: str=""):
         :param textImmediate: str, an immediate value to be translated
         :return: int, an immediate that can be worked on
         """
-
         immediate = None
         isNegative = False
         textImmediate = textImmediate.lower()  # Needed in case of 0XFF instead of 0xFF
```

Comment:
This would be really useful and eliminate the need of a function for translating immediates. We would simply need to throw out the indicator "#" before the number. I'll do some testing on this and see if it would be easy to fit in.